egs/librispeech/ASR/zipformer_adapter/train.py

```diff
@@ -1018,6 +1018,12 @@ def train_one_epoch(
         be set to 0.
     """
     model.train()
+    # set modules except adapters to eval mode
+    for name, m in model.named_modules():
+        if "adapter" in name:
+            m.training = True
```

Comment:
Why using `m.training = True` rather than `m.requires_grad_(False)` here ?

Reply:
This is only to make sure other modules are in `eval()` mode (e.g. Dropout is deactivated). We already set `p.requires_grad = False` after initializing the model.

Reply:
Ok, Thank you for your reply ~